### ISSUE TITLE
 ['Transaction 3 has invalid value (expected 0, actual 2500).'] while get_account_data

### ISSUE THREAD

**Issue by rav1112** [iotaledger/iota.lib.py#220]
What it means expected 0, actual 2500

**Comment by todofixthis** [comment]
Hey @rav1112 this message indicates that a transaction containing a signature fragment had a non-zero `value` amount.  This is not valid; see https://iota.stackexchange.com/a/936/283 for more info.

**Comment by rav1112** [comment]
Yes because I have send the some iotas. So how can we avoid this as I need all the transactions.

**Comment by minchenzz** [comment]
@rav1112  I have a same problem. Can you solve it?
